strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/TrashControllerTest.java

```diff
@@ -63,128 +55,129 @@ public void init()
 
-        assertFalse(Files.exists(repositoryTrashDir.resolve(artifactPath)),
+        assertFalse(Files.exists(artifactFileInTrash),
                     "Failed to delete artifact during a force delete operation!");
-        assertFalse(Files.exists(repositoryDir.resolve(artifactPath)),
+        assertFalse(Files.exists(repositoryDir.resolve(artifactPathStr)),
                     "Failed to delete artifact during a force delete operation!");
     }
 
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class,
+                  ArtifactManagementTestExecutionListener.class })
     @ParameterizedTest
     @ValueSource(strings = { MediaType.APPLICATION_JSON_VALUE,
                              MediaType.TEXT_PLAIN_VALUE })
-    void testDeleteArtifactAndEmptyTrashForRepository(String acceptHeader)
+    void testDeleteArtifactAndEmptyTrashForRepository(String acceptHeader,
+                                                      @MavenRepository(repositoryId = REPOSITORY_WITH_TRASH_2)
+                                                      @RepositoryAttributes(trashEnabled = true)
+                                                      Repository repository,
+                                                      @MavenTestArtifact(repositoryId = REPOSITORY_WITH_TRASH_2,
+                                                                         id = "org.carlspring.strongbox:test-artifact-to-trash",
+                                                                         versions = "1.0")
+                                                      Path artifactPath)
     {
-        String url = getContextBaseUrl() + "/api/trash/" + STORAGE0 + "/" + REPOSITORY_WITH_TRASH;
+        final String storageId = repository.getStorage().getId();
+        final String repositoryId = repository.getId();
+
+        String url = getContextBaseUrl() + "/{storageId}/{repositoryId}";
 
         ValidatableMockMvcResponse response = given().accept(acceptHeader)
                                                      .when()
-                                                     .delete(url)
+                                                     .delete(url, storageId, repositoryId)
                                                      .peek()
                                                      .then()
                                                      .statusCode(HttpStatus.OK.value());
 
-        String message = "The trash for '" + STORAGE0 + ":" + REPOSITORY_WITH_TRASH + "' was removed successfully.";
+        String message = String.format("The trash for '%s:%s' was removed successfully.", storageId, repositoryId);
         validateResponseBody(response, acceptHeader, message);
 
-        assertFalse(Files.exists(getPathToArtifactInTrash()),
-                    "Failed to empty trash for repository '" + REPOSITORY_WITH_TRASH + "'!");
+        assertFalse(Files.exists(getPathToArtifactInTrash(repository, artifactPath)),
```

Comment:
there is `RepositoryFiles.trash`, can you please use it here instead of `getPathToArtifactInTrash`?